src/logger.js

```diff
@@ -0,0 +1,21 @@
+const winston = require("winston");
+
+const logger = winston.createLogger({
+  level: "info",
+  format: winston.format.json(),
+  defaultMeta: { service: "dnslink-api" },
+  transports: [
+    new winston.transports.File({ filename: "error.log", level: "error" }),
```

Comment:
We should put logs in a `logs` directory to be able to easily mount that directory in docker-compose.